src/lib/containers/FullWidthAlert.tsx

```diff
@@ -37,6 +53,41 @@ function getIcon(variant?: string) {
   }
 }
 
+function ButtonFromConfig(props: {
+  config?: AlertButtonConfig
+  className: string
+  variant: string
+}) {
+  const { config, variant, className } = props
+  if (config) {
+    return (
+      <span // See https://github.com/wwayne/react-tooltip/issues/304
```

Comment:
Have to wrap the button in a span to show the tooltip when the button is disabled.